linkerd/task-test-util/Cargo.toml

```diff
@@ -1,17 +1,10 @@
 [package]
-name = "linkerd2-task"
+name = "linkerd2-task-test-util"
```

Comment:
WDYT about just making this a general `test-util` crate? We might want to put some other stuff that's currently in the integration test crate in there as well, in the future, if it's used in unit tests as well...